test/models/customer_test.rb

```diff
@@ -3,7 +3,33 @@
 require 'test_helper'
 
 class CustomerTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test 'Control' do
+    customer = Customer.new
+    customer.name = 'Linda Smith'
+    customer.phone = '07594548796'
```

Comment:
if this is real data for a real person... you really should change it to fake data.

Remember that this code is out on the Internet where anybody can access it. By putting this up, you're revealing personal information about somebody that a bad person could use for things like identity theft.

There are also things to think about with data protection legislation - if you use somebody's personal data, you need their permission to do so for that purpose

Reply:
That is fake data. I can change it to a name that is more obviously fake if that helps?

Reply:
no - that's fine. The important thing is that we're not leaking a real person's personal data out on to the Internet :)